.env-example

```diff
@@ -1,3 +1,6 @@
+ENTERPRISE_ENABLED=false
```

Comment:
There seems to be a mismatch between this name and the one inside of `config/secrets.yml`:

```suggestion
GITHUB_ENTERPRISE_ENABLED=false
```

Reply:
Instead, I think this might be better just detecting if a GITHUB_ENTERPRISE_HOSTNAME is present.